src/raven.js

```diff
@@ -748,35 +748,41 @@ function send(data) {
     // Set lastEventId after we know the error should actually be sent
     lastEventId = data.event_id || (data.event_id = uuid4());
 
-    makeRequest(data);
-}
+    logDebug('debug', 'Raven about to send:', data);
 
+    if (!isSetup()) return;
 
-function makeRequest(data) {
-    var img,
-        src;
+    var url = globalServer + authQueryString;
```

Comment:
This is the only part that I'm not super happy about.

I think the transport should get another `auth` key with all the pieces. If we send this as a form or XHR, I assume we wouldn't want them to be a part of the querystring, correct?

/cc @dcramer